### PR TITLE
Dominian Corvette Remap

### PR DESCRIPTION
**Pull request by lavillastrangiato**
FOR THE GODDESS, etc. Completely remaps the Dominian corvette to be in line with current mapping standards.

corvette

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!wip

**Comment by lavillastrangiato** [comment]
!review

**Comment by DreamySkrell** [comment]
image
don't put APCs or air alarms on top of shutters / blast doors / firedoors
also don't put APCs / air alarms on top of / under signs

**Comment by DreamySkrell** [comment]
image
image
Windows on thrusters, or thrusters being visible from inside of the ship, is a very old mapping style. It looks bad, and makes no sense.

**Comment by DreamySkrell** [comment]
image
scrubbers are routing waste air into a full air canister and tank
scrubbers should put waste air into empty canisters

**Comment by DreamySkrell** [comment]
image
this dock is not compliant with the docks standards

**Comment by DreamySkrell** [comment]
image
fax machine should have `department` var set (example from horizon)

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
ping me when ready for review

**Comment by lavillastrangiato** [comment]
image
The pressure tank connected to the scrubbers starts with 2500kpa and is copied from the Elyran corvette, but I have edited the starting pressure to be 0kpa so it can fill properly.

Otherwise it is fine to be reviewed. @DreamySkrell

**Comment by DreamySkrell** [comment]
image
tests fail here

**Comment by DreamySkrell** [comment]
image
the transit area thingy is only necessary for the shuttle, not the ship
it is used when a shuttle is jumping between two landmarks

image
the shuttle transit area is too small for the shuttle

**Comment by DreamySkrell** [comment]
image
do not use the `/obj/machinery/atmospherics/pipe/tank` type, it does not initialize correctly
and do not edit the pressure in the map, create a new `/obj/machinery/atmospherics/pipe/tank/air/empty` subtype and use that instead

image
I do not see any vars changed on this canister. It should be the empty yellow canister instead.

**Comment by DreamySkrell** [comment]
image
image
image
image

there is a lot of wall-mounted things overlapping

**Comment by DreamySkrell** [comment]
image
this airlock sensor is placed on the door (1), so it's not going to work properly, and sense the door atmosphere (which is static if the door is closed)
it should be on the exterior tile (2)

**Comment by DreamySkrell** [comment]
image
these should all be airless tiles

**Comment by DreamySkrell** [comment]
all the plating tiles are `/turf/simulated/floor`
they should be `/turf/simulated/floor/plating` instead

**Comment by DreamySkrell** [comment]
image
entry points on the shuttle are wrong, and it does not have an aft entry point

the ship itself does not have any entry points at all

**Comment by DreamySkrell** [comment]
image

from: <https://forums.aurorastation.org/topic/19916-dreamys-mapping-tips-and-guidelines/#comment-172873>

> Avoid "fake objects", and generally avoid changing vars like icons or icon_states. It may be fine at first, but if the icon paths or icon states change in the future, these fake objects will be broken. Prefer creating new types for things like fake props.

**Comment by DreamySkrell** [comment]
ensign's bathroom has access list(203) while all the other doors have list(212) or none

**Comment by DreamySkrell** [comment]
image
space tiles under windows

**Comment by DreamySkrell** [comment]
image
there are 7 ghostroles in total, but I see only 4 spawns on the map - can all of the roles spawn and not run out of spawn points?

**Comment by DreamySkrell** [comment]
image
apc on top of a shutter
also not connected to wire net

**Comment by DreamySkrell** [comment]
image
a lot of the lights have icon state set for no reason

image
some lights have icon state set AND inserted light
`/obj/machinery/light/colored/blue` should be used instead

**Comment by DreamySkrell** [comment]
the fax machine still needs to be fixed

**Comment by lavillastrangiato** [comment]
@DreamySkrell All of the fixes except the spawnpoints have been addressed. As people never immediately use all spawns at once, I'm not concerned about "blocked" spawns. If someone happened to be occupying a spawnpoint anyway, the next person would just spawn on top of them.

The only problem seems to be an issue with the Lammergeier but I have no idea what could be going on with it.

**Comment by DreamySkrell** [comment]
> @DreamySkrell All of the fixes except the spawnpoints have been addressed. As people never immediately use all spawns at once, I'm not concerned about "blocked" spawns. If someone happened to be occupying a spawnpoint anyway, the next person would just spawn on top of them.
> 
> The only problem seems to be an issue with the Lammergeier but I have no idea what could be going on with it.

I didn't mean if they can be blocked

I meant do they get "used up"

I don't remember and forgot to check it today, if spawn points get "used up" and removed, if someone uses the ghost spawners that use these spawn points

idk now that I think of it, I don't think they get used up

**Comment by lavillastrangiato** [comment]
> idk now that I think of it, I don't think they get used up

They don't, I've mapped in the same spawnpoints on the HICS Etna before and people spawned in those on the same tiles.

Pls take off changes required or LMK what's wrong with the cannon @DreamySkrell

**Comment by DreamySkrell** [comment]
image

entry points still wrong on the shuttle
for example on this one it is named `fore, cockpit`, but the type is `/aft`

**Comment by DreamySkrell** [comment]
image
the transit area way too big, it should only be big enough for the shuttle and ~7 tiles to the edge of it from the shuttle

image
like for example here

**Comment by lavillastrangiato** [comment]
@DreamySkrell please review my pr when convenient

**Comment by DreamySkrell** [comment]
image
missing pipe here

File: maps/away/ships/dominia/dominian_corvette/dominian_corvette_ghostroles.dm [hunk 1]
```diff
@@ -97,6 +99,7 @@
 	head = /obj/item/clothing/head/beret/dominia/priest
 	uniform = /obj/item/clothing/under/dominia/priest
 	accessory = /obj/item/clothing/accessory/poncho/dominia/red/surcoat
+	accessory = /obj/item/clothing/accessory/dominia
```
Comment (DreamySkrell) [inline maps/away/ships/dominia/dominian_corvette/dominian_corvette_ghostroles.dm:102]: member var can be set only once
only the second one will be used here

**Comment by DreamySkrell** [comment]
image
this locker has too many items, it's going to lag a bit when right clicking it

image
could add a second locker for example here, and split the items between them

also add some spare uniforms, boots, hats, whatever else the ghostroles get

**Comment by DreamySkrell** [comment]
fix the above, ping me on discord, and I'll approve

**Review by SleepyGemmy — APPROVED** [review]

**Comment by DreamySkrell** [comment]
still need spare uniforms

**Comment by DreamySkrell** [comment]
image
missing pipe here still

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**